test/plugins/scenarios/gozero/rpc_server.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by goctl. DO NOT EDIT.
+// goctl 1.7.2
+// Source: user.proto
+
+package main
```

Comment:
You have already add the auto-generate command, so I think this file can be deleted.